### PR TITLE
Updated build notes for v0.6.2

### PR DESCRIPTION
**Pull request by sandakersmann**
Updated build notes for v0.6.2

### CODE REVIEW

**Comment by peerchemist** [comment]
Let's wait until release is ready, things may change.

File: doc/build-unix.md [hunk 1]
```diff
@@ -5,39 +5,23 @@ file COPYING or http://www.opensource.org/licenses/mit-license.php.
...
-	echo "rpcuser=username" >>~/.peercoin/peercoin.conf
-	echo "rpcpassword=`head -c 32 /dev/urandom | base64`" >>~/.peercoin/peercoin.conf
-	git clone https://github.com/peercoin/peercoin
-	cd peercoin
-	git checkout v0.6.1ppc
-	cd src
-	make -f makefile.unix -j4
+	git checkout v0.6.2ppc
+	./autogen.sh
+	./configure --with-incompatible-bdb
```
Comment (hrobeers) [inline doc/build-unix.md]: This option is dangerous as it make old wallet.dat files incompatible.
Is there no way to install bdb4.8 on Debian?

Comment (sandakersmann) [reply doc/build-unix.md]: Tried this, but it did not work:

git clone https://github.com/peercoin/peercoin

cd peercoin

PEERCOIN_ROOT=$(pwd)

BDB_PREFIX="${PEERCOIN_ROOT}/db4"

mkdir -p $BDB_PREFIX

wget 'http://download.oracle.com/berkeley-db/db-4.8.30.NC.tar.gz'

echo '12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef  db-4.8.30.NC.tar.gz' | sha256sum -c

tar -xzvf db-4.8.30.NC.tar.gz

cd db-4.8.30.NC/build_unix/

../dist/configure --enable-cxx --disable-shared --with-pic --prefix=$BDB_PREFIX

make install

cd $PEERCOIN_ROOT

git checkout v0.6.2ppc

./autogen.sh

./configure LDFLAGS="-L${BDB_PREFIX}/lib/" CPPFLAGS="-I${BDB_PREFIX}/include/"

make

Comment (sandakersmann) [reply doc/build-unix.md]: I know it works with Bitcoin.

Comment (linalouise9) [reply doc/build-unix.md]: It does not work for me either, something with the configure script.

On ubuntu if I use the dbd4.8 package compiles fine. Created a separate pull request.

Comment (sandakersmann) [reply doc/build-unix.md]: Can you update the build notes for v0.6.2 in doc/build-unix.md too when you find the solution?
When you do I will close down this pull.

**Comment by linalouise9** [comment]
Actually I'm not sure what the solution is, just that there is something with the script not recognizing a custom install. Though it seems to work with Bitcoin.

To work around I installed the package

https://github.com/peercoin/peercoin/blob/develop/contrib/vagrant/install.sh#L25


#ubuntu specific for now
sudo apt-get install software-properties-common
sudo add-apt-repository -y ppa:bitcoin/bitcoin
sudo apt-get update -qq
sudo apt-get install -y libdb4.8-dev libdb4.8++-dev

**Comment by linalouise9** [comment]
Looking at the config.log errors the flags arent being passed so there is an error since the custom include and lib paths arent used. For now softlink seems to work. Updated the vagrant install.

**Pull request closed without merging**